poc/policy/histogram_with_aggregator_dp.py

```diff
@@ -0,0 +1,82 @@
+"""
+A policy that uses only Aggregator-DP to achieve central (eps, delta)-DP.
+
+Each Aggregator independently adds discrete Gaussian noise to its aggregate
+share, so that as long as at least one Aggregator is honest, the final
+aggregate result is (eps, delta)-differentially private.
+"""
+
+# Access files that live in different directories, since currently the source
+# files are not module-based.
+import os
+import sys
+# `dir_name` points to the `poc` folder in DP draft.
+dir_name = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+# Access files in `mechanism` folder.
+sys.path.append(os.path.join(dir_name, "mechanism"))
+# Access files in VDAF `poc` folder.
+sys.path.append(os.path.join(dir_name, "draft-irtf-cfrg-vdaf", "poc"))
```

Comment:
This is the exception to the above comment: I don't think there's a better way to to include sources from a submodule.